e2e/tests/003-network-topo.tmpl

```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   nodes:
     srl: 
+      address: $LEAF_IP:57400
       provider: srl.nokia.com
```

Comment:
curious if this is something nokia specific?

Reply:
no this is relevant for any network device. but the port might be different

Reply:
> no this is relevant for any network device. but the port might be different

Apologies, I mean the `provider: srl.nokia.com`

Reply:
@henderiw 
```
provider: srl.nokia.com
```
That I guess should be changed to something more generic if that needs to be there as an configuration example as @vjayaramrh pointed out